cmd/root.go

```diff
@@ -186,12 +186,8 @@ func ensureToken(ctx context.Context, requiredScopes []string) (context.Context,
 	}
 
 	// Check to see if the URL is secure
-	gatewayUrl := viper.GetString("gateway-url")
```

Comment:
Does the removal of this `gateway-url` flag mean it's disused? If so should we remove it from the CLI flags too so it doesn't show in the help

Reply:
--gateway-url is used elsewhere, so it needs to stay.